dataland-frontend/cypress.config.ts

```diff
@@ -12,7 +12,9 @@ export default defineConfig({
         prepopulate_timeout_s: 180,
         short_timeout_in_ms: 10000,
         medium_timeout_in_ms: 30000,
-        long_timeout_in_ms: 120000,
+        long_timeout_in_ms: 60000,
```

Comment:
should the long timeout be explicity changed or did this survive temporary changes?

Reply:
@emanueldfine @fsoffel do you guys know why this was changed?
My assumption would be that after the refactoring of the tests less time to wait is necessary

Reply:
@emanueldfine

Reply:
Changed this simply because 120.000 makes no sense at all.
Even 60.000 or 30.000 is pretty heavy and doesn't make much sense.
No cypess test in the world should require timeout lengths like these.
I simply changed it to 60.000 be a "little" more okeyish.